platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/sensors/Bme280Generator.xtend

```diff
@@ -117,7 +117,7 @@ class Bme280Generator extends AbstractSystemResourceGenerator {
         return switch(modalityName) {
             case 'temperature': codeFragmentProvider.create('''«dataVariable».temperature''')
             case 'pressure': codeFragmentProvider.create('''«dataVariable».pressure''')
-            case 'humidity': codeFragmentProvider.create('''((float)«dataVariable».humidity / 1024.0f)''')
+            case 'humidity': codeFragmentProvider.create('''((float)«dataVariable».humidity)''')
```

Comment:
Since `humidity` is already in %rH and an `uint32` we should change the modality to `uint32` and remove the `fixed_point` modality. In that case please also update the documentation in `/website/site/content/platforms/xdk110.md`.

Reply:
Another possibility is to change the generated C code so it exposes some kind of sub-percentage information.

Reply:
updated with only humidity in  %rH